src/marine/smap_sss2ioda.py

```diff
@@ -134,7 +134,7 @@ def main():
 
     parser = argparse.ArgumentParser(
         description=(
-            'Read JPL/RSS SMAP sea surface salinity (SSS) file(s) and convert'
+            'Read JPL/RSS SMAP sea surface seaSurfaceSalinity (SSS) file(s) and convert'
```

Comment:
@CoryMartin-NOAA  @travissluka   This is the line that I saw the strange double-wording.  It was a false alarm on the other spot as I got confused in reading the email that Github sent when Travis made the suggestion.  So please fix this one as well @PraveenKumar-NOAA

Reply:
@CoryMartin-NOAA @gthompsnJCSDA  @travissluka @guillaumevernieres: fixed the strange double wording in both , smos and smap, the converters and also added Marine/hgodas_sst2ioda.py converter to this PR.